hw/xbox/nv2a/nv2a_psh.c

```diff
@@ -27,16 +27,16 @@
 
 #include "qemu/osdep.h"
 
+#include <stdbool.h>
+#include <stdint.h>
```

Comment:
Note to self: Pausing review at this point (done commit-by-commit, so please don't force-push). I feel this needs a lot of concentration. Will continue later.